packages/components/src/color-picker/index.native.js

```diff
@@ -101,7 +101,8 @@ function ColorPicker( {
 		if ( onHandleHardwareButtonPress ) {
 			onHandleHardwareButtonPress( onButtonPress );
 		}
-	}, [] );
+	} );
+	useEffect( bottomSheetInitializer, [ bottomSheetInitializer ] );
```

Comment:
Hmm... I kind of prefer adding an eslint-disable with a TODO comment here instead of legitimizing the [lifecycle-reliant approach](https://epicreact.dev/myths-about-useeffect/) of the code. I'm not sure how this component is consumed (or if it has third-party consumers), but it seems to be setting some potential bugs in stone. With an eslint-disable and TODO, we can at least flag that it's problematic. What do you think? (cc @geriux)

Reply:
I've gone ahead and changed to the eslint-disable + TODO comment in aa7a41b.

BTW, when I first worked on this I did (for the first time) run the iOS emulator to test and maybe see if I could better understand what this effect was for but I couldn't even find a `ColorPicker` to manipulate 🤷‍♂️.